Project.toml

```diff
@@ -18,7 +18,7 @@ TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
 ArgParse = "1.1.4"
 Folds = "0.2.8"  
 HDF5 = "0.16.15"
-IceFloeTracker = "0.2.1"
```

Comment:
🤩

Reply:
That probably belongs in a separate PR? That is parameter of feature extraction.